examples/contracts/src/lib.rs

```diff
@@ -382,11 +409,25 @@ mod tests {
             // you don't need any funds to read state
             let balance = contract_methods
                 .get_balance(contract_id, AssetId::zeroed())
-                .simulate(Execution::StateReadOnly)
+                .simulate(Execution::state_read_only())
                 .await?
                 .value;
             // ANCHOR_END: simulate_read_state
         }
+        {
+            let provider = wallet.provider();
+            provider.produce_blocks(2, None).await?;
+            let block_height = provider.latest_block_height().await?;
+            let contract_id = contract_id.clone();
+
+            // ANCHOR: simulate_read_state_at_height
```

Comment:
Nice, didn't know about these until know: https://rust-lang.github.io/mdBook/format/mdbook.html#including-portions-of-a-file